extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/ranger/IcebergCatalogRangerSparkExtensionSuite.scala

```diff
@@ -106,12 +113,10 @@ class IcebergCatalogRangerSparkExtensionSuite extends RangerSparkExtensionSuite
     withSingleCallEnabled {
       val e2 = intercept[AccessControlException](
         doAs(
-          someone,
+          bob,
           sql(mergeIntoSql)))
       assert(e2.getMessage.contains(s"does not have" +
-        s" [select] privilege" +
-        s" on [$namespace1/$table1/id,$namespace1/table1/name,$namespace1/$table1/city]," +
-        s" [update] privilege on [$namespace1/$outputTable1]"))
+        s" [update] privilege on [$bobNamespace/$bobSelectTable]"))
     }
```

Comment:
`withSingleCallEnabled` is for listing all the unsatisfied privileges in the exception. We still need a negative case for `someone` who is anonymous and has no permission on the source and target table at all.

Reply:
There is subQuery in the `mergeIntoSql` execution plan in Spark 3.5. When applying the `OptimizeSubqueries` rule, it returns no select permission and no update. If we want to verify `withSingleCallEnabled` I think we need to use other statements, like: `insert into ... select ...`

Reply:
> If we want to verify withSingleCallEnabled I think we need to use other statements, like: insert into ... select ...

It is covered in: 

```
"[KYUUBI #3371] support throws all disallowed privileges in exception"
```

Reply:
Let's bring back the single call ut for `someone` with no permissions to ensure end-to-end results.